tests/components/sharkiq/test_config_flow.py

```diff
@@ -0,0 +1,147 @@
+"""Test the Shark IQ config flow."""
+import aiohttp
+from sharkiqpy import SharkIqAuthError
+
+from homeassistant import config_entries, setup
+from homeassistant.components.sharkiq.const import DOMAIN
+
+from .const import CONFIG, TEST_PASSWORD, TEST_USERNAME, UNIQUE_ID
+
+from tests.async_mock import MagicMock, PropertyMock, patch
+from tests.common import MockConfigEntry
+
+
+def _create_mocked_ayla(connect=None):
+    """Create a mocked AylaApi object."""
+    mocked_ayla = MagicMock()
+    type(mocked_ayla).sign_in = PropertyMock(side_effect=connect)
+    type(mocked_ayla).async_sign_in = PropertyMock(side_effect=connect)
+    return mocked_ayla
+
+
+async def test_form(hass):
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == "form"
+    assert result["errors"] == {}
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6090606)